modules/agar/CHANGELOG.md

```diff
@@ -8,6 +8,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ### Improved
 - Exposed the `ArrayAssert` and `StringAssert` types.
+- Exposed the `KeyPressAdt` type. #TINY-8952
```

Comment:
Is this really something we need to expose. Looking at the test the `pSendKeysToActiveInput` seems to be called with only a text or a space key. Maybe ReadKeys should instead expose two functions that does those things? Since we are moving away from this old adt style we should not use it further if we don't need to. Could we hide this?

Reply:
The main problem is while we don't have an alternative, not exposing this prevents us from having any type safety when creating wrapper functions for RealKeys. We can provide the changes in agar if it's really preferred, but it seemed likely that just replacing KeyPressAdt in one go would be easier and more focused, rather than having yet another focus in this PR.